src/wavesurfer.js

```diff
@@ -721,7 +724,12 @@ export default class WaveSurfer extends util.Observer {
             this.fireEvent('finish');
         });
         this.backend.on('play', () => this.fireEvent('play'));
-        this.backend.on('pause', () => this.fireEvent('pause'));
+        this.backend.on('pause', () => {
+            if ( this.params.returnToPlayhead ) {
```

Comment:
adding more complexity to these event triggers is something we should prevent at all times.

Reply:
the `pause()` method then?

Reply:
let's give it a try.